pkgs/development/libraries/libprelude/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, fetchurl
+, pkgconfig
+, bison
+, flex
+, gnutls
+, lua
+, perl
+, python
+, ruby
+, swig
+}:
+
+let
+
+in
+stdenv.mkDerivation rec {
+  name = "libprelude-${version}";
+  version = "4.1.0";
+
+  src = fetchurl {
+    url = "https://www.prelude-siem.org/attachments/download/831/libprelude-4.1.0.tar.gz";
+    sha256 = "0bana4wq4n64i90pxjljksi34yvvja8pib86qq1nj4y45zp5pvi1";
+  };
+
+  nativeBuildInputs = [
+    bison
+    flex
+    pkgconfig
+    swig
+  ];
+
+  buildInputs = [
+    gnutls
+  ];
+
+  propagatedBuildInputs = [
```

Comment:
Should all of these really be propagated?
I think these should probably be separate packages in their respective languages (eg python module in `pkgs/development/python-modules`).